### ISSUE TITLE
Accents of countries in which English is not an official language

### ISSUE THREAD

**Issue by a3nm** [mozilla/voice-web#242]
In the list of English accents, there are only accents for countries where English is a native language, but there are no foreigner accents (e.g., German, French...). I would propose adding them, or adding a more general "non-native" entry.

(Of course, maybe the project doesn't want to collect voice samples from non-native speakers, but then this should be indicated beforehand on the landing page.)

**Comment by kdavis-mozilla** [comment]
We want non-native speakers _and_ this lack of "English with a German Accent" selection is a conscious choice. 

If, for example, a German moves to Great Britain and learns English, then they would speak British English but with a German accent. They would not speak "English with a German accent", as "English with a German accent" does not really exist because there is no generic "English" accent.

So we want to force a choice on the users so we can then collect the diversity of "British English" and be able to account for things like people who grew up in Great Britain, people who moved there and learned English there, people whose English teacher had a British English accent...

The logic of it from a speech recognition point of view is that one deploys a speech recognition engine to say Great Britain. It has to deal with Germans that have moved there and learned English, Poles that have moved there and learned English, people that grew up there...

**Comment by a3nm** [comment]
OK, thanks for this clarification. I understand your point.

That said, maybe a clarification like this should be made near the "English accent" field on the profile (or maybe in the FAQ), because otherwise the options in the list give the impression that people outside of English-speaking countries are not supposed to contribute.

**Comment by kdavis-mozilla** [comment]
Good point!

**Comment by Grovespaz** [comment]
I'm an Dutch person and English speaker. My accent is probably mostly American due to the influence of US media, but I communicate with a whole bunch of people with different accents online through voice chat. About half of them are British and I'm sure I pick up parts of their accents.
I've never lived in the US, I'm really not a good example of an American English accent nor a British one.

At best, my accent is an amalgam with different influences. Are you sure you want me dirtying your data? Perhaps a simple boolean "I'm not a native speaker" could make a world of difference to some of the folks using this data. A bit of granularity probably wouldn't hurt.

**Comment by mikehenrty** [comment]
> At best, my accent is an amalgam with different influences. Are you sure you want me dirtying your data? 

We definitely want your data. Sounds like a little description on the profile page explaining the situation would go a long way. Long story short, we want anyone who can speak English contributing!

**Comment by mikehenrty** [comment]
changing to p1 based on the amount of feedback here and on hackernews/reddit/other bugs.

**Comment by edunham** [comment]
IMO the actual region question might be best answered by asking "which regional accents have strongly influenced your English accent" or something, and letting the user check as many as they like. This addresses the situation of people who learn English from American TV shows but have never been to the US, and me as a US resident who picked up one parent's NZ accent, and people who move between regions in childhoood, and so forth.

**Comment by achimkoh** [comment]
>Long story short, we want anyone who can speak English contributing!

Thanks for clarifying. It seems that this is already being taken into account by the team, but I just wanted reiterate on @a3nm 's comment that

> otherwise the options in the list give the impression that people outside of English-speaking countries are not supposed to contribute.

Because not only was this exactly my impression when I clicked on the dropdown menu, which was discouraging, but I also thought that the project is geared towards somewhat _standard_ models of regional accents, which is to be trained using "good example[s]" (as @Grovespaz mentioned) from people who have a mastery of those accents. 

However, it seems that the project actually aims at gathering diverse data from real world people, however "imperfect" their accent might be. I think it is a great approach and its inclusivity should be made clearer, especially to people providing their voice.

**Comment by raphaeldisanto** [comment]
> IMO the actual region question might be best answered by asking "which regional accents have strongly influenced your English accent" or something, and letting the user check as many as they like. 
I love this suggestion. I don't know how feasible it would be to retroactively change the entries of people who have submitted data already, but it sounds to me like a far more inclusive way of gathering accent data. In today's multicultural society, there are many people who have all manner of accent influences. 

I'm Cambodian born, raised in Wales, but have lived in the US for the last 25% of my life. My accent is all over the place. British people think I sound American. Americans think I sound British. No one thinks I sound Welsh, unless I decide to fake the accent.

**Comment by rugk** [comment]
In https://github.com/mozilla/voice-web/issues/273 there was this comment:
> Probably non-native speakers should always specify their native language(s) and/or native linguistic region, and this information should be provided to judges. 

This comment was made at the beginning of the issue. But in fact there is, when you're not living in a country where English is speaken officially (i.e. where it is a national language or so) or just have the accent from there, you have no way to select this. There are only few regions covered.

> Long story short, we want anyone who can speak English contributing!

The thing is, many people can speak English, but few of them (I think) live . English is speaken on the whole world (as a "lingua franca") and so you are currently only "allowing" a small part of potential speakers to be a part as IMHO when some people do not find their own accident in the list, they think they would only "pollute" the data without having a way to later filter it out. At least that's what I thought.

Basically there are many areas missing, such as continental Europe, northern Africa and Asia, and South America.

If you need a way to verify whether non-native speakers speak English well-enough, just do something like this: https://github.com/mozilla/voice-web/issues/322

**Comment by HuBandiT** [comment]
I am a non-native English speaker, and I would like to strongly vote for recording the native language of the speaker (as well as optionally the English standard they are trying to mimic) with the data.

Reasoning:

Non-native English speakers will often have quite strong differences from "standard English" (regardless of which standard is meant at the moment). These are usually systematic, and very strongly correlate with and are predicted by their native language. E.g. (in my own case) v-w confusion, d-th confusion, bad-bed confusion, etc.

If the data for non-natives is tagged with native language from the outset, that will enable us to try to learn specific native-language influences much more precisely (more focused, sharper gaussians, and speaker native language could become one state variable in the HMM that has a low probability to change - either or both should result in more accuracy), than if we would not tag with native language, forcing us to have to learn from a melting pot of unknown languages, muddling the acoustic model learning, worsening accuracy.

**Comment by devurandom** [comment]
I validated quite a few samples, and I can tell you that I heard a few accents that I am 100% positive are German or others being somewhere from Africa with some hint of French. But I would have no clue whether they were more British or American or Australian (or anything else) and I somewhat doubt that an ML-system would be able to put them into any such category.

I know many people who try to imitate a British accent, but at the same time their accent contains lots of influences from the US, e.g. through movies. Despite all their efforts, I doubt anyone would be able to hear that through their gross German accent. Others have never been in an English speaking country and don't watch movies in English or listen to radio shows -- how would you categorise their accent? I couldn't tell, except saying its definitely "German". I would assume Germans are no strange exception and our fellow French(wo)men, Italians or Spaniards, and people from any other country are no different.

That said, the German or African accent in the samples was strongly dominating and my guess would be that trying to force them into categories of native accents is going to mess up your data. IANAL - I Am Not A Linguist ;)

**Comment by a3nm** [comment]
Thanks for fixing this! :)

-- 
Antoine Amarilli

**Comment by rugk** [comment]
However, still only a box "Other accents"… https://github.com/mozilla/voice-web/commit/99e7d7b4c21d5a0890823f97067559798536ebf8#commitcomment-23351511

**Comment by mikehenrty** [comment]
Sorry all, had to re-open this issue as it didn't address the speech team's concerns as I initially thought. We still have the faq change, but as for the drop-down, we'll need to re-examine it.

**Comment by Kakurady** [comment]
Perhaps "Non-native accent" could be a checkbox instead of a list item? 

Then, for example, a German moves to Great Britain and learns English, would select "British English" with "☑ Non-native accent".

(Although this fails to capture the influence of the speaker's native language(s). For example, in spoken Standard Chinese there is no /θ/ or /ð/ sound, and I used to substitute with /t͡s/ or /t/. I understand this information isn't very useful now - but this dataset could be used with methods & applications that are yet to be created.)

**Comment by iveskins** [comment]
How about asking 
"When did you start learning English?"
"List all native language(s) learned from birth (if not listed, choose 'other'):"
"Your primary language(s) now (if not listed, choose 'other')"
like 
http://archive.gameswithwords.org/WhichEnglish/

**Comment by mkristian** [comment]
in short: I was unsure whether or not I can contribute with my melange of british, american and indian english. I just left the `accent` drop down unchecked but still was unsure whether I shall add my voice or not. a little hint that everyone is most welcome whether native speaker or not, needs to be added. I did speak only a handful sentences due to this uncertainty.

now I know I can help recording more sentences . . 

just a little notice somewhere on the main page: any native and none-native speaker is welcome would help IMO

**Comment by Cromefire** [comment]
@Kakurady this approach is also not satisfying, because most not native speakers speak at least a mix of British and American English. It's not only about the few who moved to the UK, but also about for example about people who never lived in an English-speaking country.
I think at the current time the best choice is to make an "other" option and later do it using something like primary and secondary language.

**Comment by odinho** [comment]
`<Primary language>`-English, such a neat solution @iveskins. So for me that'd be `Norwegian-English`. When you choose `non-native` English, you get to choose your primary language or the one most 'coloring' your English. Suddenly you get French-English, German-English, Spanish-English, Polish-English and all the rest we very easily can tell by listening to people.

**Comment by nukeador** [comment]
We are going to open a conversation on discourse about accents in the coming days with a proposal.

Please, add your feedback there so we can have a unified approach for all languages

https://discourse.mozilla.org/c/voice

Thanks for your contributions!

**Comment by nukeador** [comment]
Hello,

We have opened a call for feedback about Common Voice's languages and accents strategy. I would like to invite you all to the conversation and add your input before May 26th.

Thanks so much for your feedback!

https://discourse.mozilla.org/t/feedback-needed-languages-and-accents-strategy/40352
